addons/account/models/account_journal_dashboard.py

```diff
@@ -654,31 +654,29 @@ def _fill_general_dashboard_data(self, dashboard_data):
         general_journals = self.filtered(lambda journal: journal.type == 'general')
         if not general_journals:
             return
-        to_check_vals = {
-            journal.id: (amount_total_signed_sum, count)
-            for journal, amount_total_signed_sum, count in self.env['account.move']._read_group(
+        journals_with_aliases_ids = general_journals.filtered(lambda journal: journal.alias_domain_id and journal.alias_name).ids
+        draft_vals = {
+            journal.id: count
+            for journal, count in self.env['account.move']._read_group(
                 domain=[
                     *self.env['account.move']._check_company_domain(self.env.companies),
-                    ('journal_id', 'in', general_journals.ids),
-                    ('checked', '=', False),
-                    ('state', '=', 'posted'),
+                    ('journal_id', 'in', journals_with_aliases_ids),
+                    ('state', '=', 'draft'),
                 ],
                 groupby=['journal_id'],
-                aggregates=['amount_total_signed:sum', '__count'],
+                aggregates=['__count'],
             )
         }
         for journal in general_journals:
-            currency = journal.currency_id or self.env['res.currency'].browse(journal.company_id.sudo().currency_id.id)
-            amount_total_signed_sum, count = to_check_vals.get(journal.id, (0, 0))
+            draft_count = draft_vals.get(journal.id, 0) if journal.id in journals_with_aliases_ids else 0
```

Comment:
I think only journals in `journals_with_aliases_ids` are in `draft_vals`?
```suggestion
            draft_count = draft_vals.get(journal.id, 0)
```
I think we could also get rid of the variable and assign the value directly to `'number_draft'` later.